packages/blockchain-tools/src/contracts/soroban/credits/src/tests/test_credits.rs

```diff
@@ -48,8 +48,21 @@ fn test_donate() {
   // Donate
   test.credit.donate(&donor, &100_000_000);
 
-  assert_eq!(test.credit.getBalance(), 90_000_000); // amount - vendor fees
-  assert_eq!(test.credit.getContractXLMBalance(), 90_000_000);
+  /*
+    1. first swap for XLM <-> USDC with 10_000 XLM
```

Comment:
1 XLM is 10_000_000 in stroops (the integer unit), so the swap in this test is done with 9 XLM as input